packages/indexer/lib/index.js

```diff
@@ -91,6 +101,23 @@ class Indexer {
   collector({ lock = null, type = null, argsLen = -1, data = "0x" } = {}) {
     return new CellCollector(this, { lock, type, argsLen, data });
   }
+
+  subscribe({ lock = null, type = null } = {}) {
```

Comment:
Is there any blocker from supporting the full [QueryOptions](https://github.com/nervosnetwork/lumos/blob/develop/packages/base/index.d.ts#L85) interface? I do believe we can support full QueryOptions interface if possible.

Reply:
Yes, it's feasible to support range parameters(`fromBlock|toBlock`) and args prefix parameters(`argsLen|data`), and the `skip` parameter doesn't make any sense here. But I doubt about the actual use cases of those fine-grained features when monitoring a script's update, and try to avoid adding extra complexity if it's unnecessary.

Reply:
We could ignore those who does not make sense, like `skip`, but I do believe `argsLen`, `data`, and arguable `fromBlock` can be quite helpful for building queries. For example, supporting `multisig` script will naturally need `argsLen`, while Nervos DAO monitoring will require querying `data`. So I do think we should use full `QueryOptions` here.

Reply:
Ok, I'll add support for `argsLen|data|fromBlock` in the current version.